lookup_plugins/bitwarden.py

```diff
@@ -142,6 +146,15 @@ def run(self, terms, variables=None, **kwargs):
                 values.append(bw.get_custom_field(term, field))
             elif field == 'notes':
                 values.append(bw.get_notes(term))
+            if kwargs.get('attachments'):
```

Comment:
@c0sco && @Maltazar this line breaks custom fields support. Should be `elif` not `if`.

Reply:
Good find! I've opened #16 and will fix this shortly.